### DISCUSSION TITLE
Github Desktop UI font size scaling

### DISCUSSION THREAD

**Discussion by arnvidhr** [community/community#78691] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, how to enable font scaling in Github Desktop application according to the Windows screen DPI settings, or at least to choose the font size?

After so much decades of coding and gaming, my eyesight is not very good and it is difficult to see such small text. It feels worrying for future because there is no way to change font size or scale. And it seems that the application is not affected by specific Microsoft Windows setting.

This is something for your consideration.

**Comment by DeadGolden0** [comment]
Hi @arnvidhr ,

The GitHub Desktop application doesn't have a font setting, but it does support zooming in and out, which will in turn increase or decrease the font size. See the View menu or try ``Ctrl +`` and ``Ctrl -`` .

Do not hesitate to mark the post as 'Answered' if my response suits you.

I hope my message is useful to you! Have a nice day!

**Comment by tyehuoPHD** [comment]
After i click on the zoom in or zoom out in `view`  menu, nothing happens..

**Comment by costas-andriotis** [comment]
If you are mildly familiar with CSS, open Developer Tools -> Sources and check the location of the file **_renderer.css_**. You can try editing this file to suit your needs. For example, I'm using a UHD monitor and I also find some fonts very small, so I added the following at the very end of the file:

`:root { --font-family-sans-serif: 'Noto Sans Condensed'; --font-family-monospace: 'Noto Sans Mono ExtraCondensed'; --font-size: 14px; --font-size-sm: 12px; font-weight: 500;}`

You could try other customizations at your risk, if you like, keeping in mind they probably won't survive updates.

**Comment by xyzzy529** [comment]
I love GitHub.com ... have for 17 years. I was 17 years younger then ...
Did not have cataracts, and could see any size.
Now it would be better for the site to have a font size in the Profile setting so it would be permanent and not a pain each time I look at a page.

Also, there is this: https://adasitecompliance.com/accessible-fonts/
The Americans with Disabilities Act (ASA) requires websites to comply with this sort of accessibility requirements.

@GitHub ... Please fix this before someone sues you over it and the Legal fees cost more than a few lines of code would.

**Reply by panda667** [reply]
I totally agree! I am young but have low vision. The standard github font is really tiny and is not affected by browser fixed-width font settings.
